src/wp-content/themes/twentytwenty/assets/css/editor-style-block-rtl.css

```diff
@@ -206,20 +206,24 @@ Inter variable font. Usage:
 /* GENERAL COLORS */
 
 .has-black-background-color {
+	--wp--preset--color--black: #000;
```

Comment:
Apparently, in addition to bringing back the regular CSS Properties, https://github.com/WordPress/wordpress-develop/pull/2154 missed generating the CSS Custom Properties for these values. This PR also fixes it.